types/gulp-modernizr/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for gulp-modernizr 1.0
+// Project: https://github.com/doctyper/gulp-modernizr
+// Definitions by: remisery <https://github.com/remisery>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function GulpModernizr(fileOrParams?: string | GulpModernizr.Params, parames?: GulpModernizr.Params): any;
```

Comment:
I think you dont want people to send you two parameters as Options correct? 
So you want to write this as overload functions as:
```TypeScript
declare function GulpModernizr(fileOrParams?: GulpModernizr.Params): any;
declare function GulpModernizr(fileOrParams?: string, params?: GulpModernizr.Params): any;
```